### DISCUSSION TITLE
The Reminder Suddenly Weird

### DISCUSSION THREAD

**Discussion by Aldis97** [iSoron/uhabits#910] [Help & FAQ]
The reminder used to appears automatically and stayed on the top bar until I checked it. However it suddenly 1. did not show in time unless I open the apps 2. after appears on the top bar, it will disappear if I close the app whether I check it or not even though I already set the notification sticky. I already reinstall it and it still behave the same. What's wrong?

**Comment by iSoron** [comment]
Hi @Aldis97, it looks to me like your phone is killing the app automatically. Could you please let us know: (i) what is your phone, (ii) what is the phone operating system (e.g. Android 10), and (iii) what is the app version? I would also suggest changing your phone settings so that it disables battery optimizations for Loop Habit Tracker. [This website shows how to do this in different phones](https://dontkillmyapp.com/).

**Reply by Aldis97** [reply]
Thank you! The apps already doing fine. It seems the setting was changed due the phone frozen a while ago.

**Reply by Aldis97** [reply]
> Hi @Aldis97, it looks to me like your phone is killing the app automatically. Could you please let us know: (i) what is your phone, (ii) what is the phone operating system (e.g. Android 10), and (iii) what is the app version? I would also suggest changing your phone settings so that it disables battery optimizations for Loop Habit Tracker. [This website shows how to do this in different phones](https://dontkillmyapp.com/).

Hello, sorry to bother again. 
The reminders are fine after I change the battery setting, however I notice that the reminders only stay for a while? Usually if I'm not check it or swap it right, the reminders will stay there even the earliest one. I also notice when I'm using app that turned off navigation or click another app from top bar notification, the reminders will disappears. I do not accidentally remove them. Before, it stays there for long time until I do something. If I don't check my phone, I missed the reminders. I wonder what's different right now.

I'm using XiaoMi Redmi 9, MIUI 12, Android 10, Apps version 1.8.12.
